src/rest-server/deploy/rest-server.yaml.template

```diff
@@ -29,7 +29,7 @@ spec:
       labels:
         app: rest-server
     spec:
-      hostNetwork: false
+      hostNetwork: true
```

Comment:
could you fix the cni issue instead? it's not a good practice to use `hostNetwork`

Reply:
CNI issue is not easy to fix and we don't have a solution yet. Waiting for @ydye investigation.